src/ui/MultiSelect/MultiSelect.tsx

```diff
@@ -80,7 +78,7 @@ const MultiSelect: React.FC<MultiSelectProps> = ({
     <div ref={refOptions}>
       <InputBox onClick={() => setIsOpen((prev) => !prev)} $isOpen={isOpen}>
         <div className="input-box-wrapper">
-          <span className="span-question">Qué sevicio requiere? *</span>
+          <span className="span-question">¿Qué sevicio requiere? *</span>
```

Comment:
this shouldn't have the star (mandatory field) and it should be setted when the component is used, not as default for the component because we may use this component not only for services